src/Avalonia.Themes.Default/ComboBox.xaml

```diff
@@ -39,7 +39,7 @@
                    StaysOpen="False">
               <Border BorderBrush="{DynamicResource ThemeBorderMidBrush}"
                       BorderThickness="1">
-                  <AdornerDecorator Margin="-1 -1 0 0">
+                  <VisualLayerManager IsPopup="True" Margin="-1 -1 0 0">
```

Comment:
Now that `PopupHost` is a templated control too, we can probably move this into the template?